src/databases/Sqlite.js

```diff
@@ -0,0 +1,32 @@
+const { db } = require("./databases");
+
+class Sqlite {
+  constructor(connection) {
+    this.connection = connection;
+  }
+
+  getConnection() {
+    return this.connection;
+  }
+
+  prepare(type, query, params) {
+    //console.log("prepare:        type: " + type + ", query: " + query + ", params: " + params);
+    if (type === 'get') {
+      return this.connection.prepare(query).get(...params);
+    } else if (type === 'run') {
+      this.connection.prepare(query).run(...params);
+    } else if (type === 'all') {
+      return this.connection.prepare(query).all(...params); 
+    } else {
+      console.log('returning undefined...')
+      console.log("prepare:        type: " + type + ", query: " + query + ", params: " + params);
```

Comment:
Could these be done only on debug?

Reply:
They surely could, happy to change that (i didn't because i wanted to keep track after deploying it myself)